compiler/luci/pass/include/luci/CircleOptimizer.h

```diff
@@ -47,6 +47,7 @@ class CircleOptimizer final
       FuseMulWithConv,
       FuseMulWithDiv,
       FuseTransposeWithMean,
+      ResolveCustomOp,
```

Comment:
The overall description of the new pass is as follows.

- The discussion of the New pass, `ResovleCustomOpPass`, came from here.
-> https://github.com/Samsung/ONE/issues/10784#issuecomment-1797862821
- This new pass is a pass that changes op, which was previously supported by `custom` op, 
when it can now be supported by `built-in` op.

Reply:
Q) are you going to support conversion of all custom ops to built-in ops?

Reply:
@seanshpark 

> Q) are you going to support conversion of all custom ops to built-in ops?

No. This pass will only support specific `custom` op that currently come into the builtin op. 
Until now, only BroadcastTo op is applicable to this case. :)

Reply:
There exist
![image](https://github.com/Samsung/ONE/assets/4616940/06231f16-0483-4c63-91c5-32fe5fe8ed8e)

Your new pass name looks like it will absorb all existing ones.

Reply:
@seanshpark 

Oh... Thank you for letting me know :)
Could you recommend the name of new pass ?? 😀

Reply:
ResolveExCustomOp..?? AdaptExCustomOp..??

Reply:
@mhs4670go 
> ResolveExCustomOp

I think it is a good.
Thank you for your comments :)